manifest/oci.go

```diff
@@ -32,31 +31,12 @@ type OCI1 struct {
 	imgspecv1.Manifest
 }
 
-// SupportedOCI1MediaType checks if the specified string is a supported OCI1 media type.
-func SupportedOCI1MediaType(m string) error {
```

Comment:
Removing a public function would be an API break.

Please keep it around, _including tests_, with a https://github.com/golang/go/wiki/Deprecated comment (saying something to the effect that blindly rejecting unknown MIME types when the consumer does not need to _process_ the input just reduces interoperability with no benefit, and that this function does not check that the media type is appropriate for any specific purpose, so it’s not all that useful for validation anyway).

Reply:
Adding this check was also an API break, but ok :). I've re-added it.